docs/ProductImages.md

```diff
@@ -112,7 +117,9 @@ The following table shows the props allowed by `product-images.high-quality-imag
 | `imageSizes` | `[number]` | Image size(s) (in `px`) to be used in the image's [`srcset` HTML attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images). If no value is passed to this prop, the `srcset` will use the image original size.  | `undefined` |
 | `maxSize` | `number` | Image maximum size (in `px`) for rendering regardless of the screen size. Notice that this prop only works if you also declare the `imageSizes` prop. | `4096` |
 
-You also can wrap the main slider with another layout app, this can be useful when you will pass a floating badge over the image. The main slider is exported by `product-images.main-swiper`. Here an implementation example:
```

Comment:
Adding this part to the advanced configuration section is a good alternative @IncognitaDev . My suggestions are related to how we present the information in the documentation. 

In the introduction of the advanced configuration section, we only mention using modal zoom, and I believe the `product-images.main-swiper` is worth mention and also a subcategory for it as well. 

In the `advanced configuration` first paragraph, I would present what the person will see in this section, and then I would add two sub-sections: one for modal zoom and another for wrapping main slider.

Reply:
Sorry, I see that you already make some changes, is there any change that still needs to be done?

Reply:
No worries! LGTM 🚀